recipe-server/normandy/recipes/tests/api/v1/test_api.py

```diff
@@ -572,6 +573,32 @@ def test_list_filter_action(self, api_client):
             assert res.status_code == 200
             assert res.data == []
 
+        def test_order_last_updated(self, api_client):
+            now = datetime.now()
+            yesterday = now - timedelta(days=1)
+            r1 = RecipeFactory(updated=yesterday)
+            r2 = RecipeFactory(updated=now)
+
+            res = api_client.get(f'/api/v2/recipe/?ordering=last_updated')
```

Comment:
Why an f-string?

Reply:
This was a copy and paste failure, the test that I duplicated actually needed the f-string for including a recipe id. Will fix.